Client.php

```diff
@@ -54,49 +54,70 @@ class Client extends Connection
      *
      * @const int
      */
-    const ASYNCHRONOUS      = STREAM_CLIENT_ASYNC_CONNECT;
+    const ASYNCHRONOUS        = STREAM_CLIENT_ASYNC_CONNECT;
 
     /**
      * Open client socket connection.
      *
      * @const int
      */
-    const CONNECT           = STREAM_CLIENT_CONNECT;
+    const CONNECT             = STREAM_CLIENT_CONNECT;
 
     /**
      * Client socket should remain persistent between page loads.
      *
      * @const int
      */
-    const PERSISTENT        = STREAM_CLIENT_PERSISTENT;
+    const PERSISTENT          = STREAM_CLIENT_PERSISTENT;
 
     /**
      * Encryption: SSLv2.
      *
      * @const int
      */
-    const ENCRYPTION_SSLv2  = STREAM_CRYPTO_METHOD_SSLv2_CLIENT;
+    const ENCRYPTION_SSLv2    = STREAM_CRYPTO_METHOD_SSLv2_CLIENT;
 
     /**
      * Encryption: SSLv3.
      *
      * @const int
      */
-    const ENCRYPTION_SSLv3  = STREAM_CRYPTO_METHOD_SSLv3_CLIENT;
+    const ENCRYPTION_SSLv3    = STREAM_CRYPTO_METHOD_SSLv3_CLIENT;
 
     /**
      * Encryption: SSLv2.3.
      *
      * @const int
      */
-    const ENCRYPTION_SSLv23 = STREAM_CRYPTO_METHOD_SSLv23_CLIENT;
+    const ENCRYPTION_SSLv23   = STREAM_CRYPTO_METHOD_SSLv23_CLIENT;
 
     /**
      * Encryption: TLS.
      *
      * @const int
      */
-    const ENCRYPTION_TLS    = STREAM_CRYPTO_METHOD_TLS_CLIENT;
+    const ENCRYPTION_TLS      = STREAM_CRYPTO_METHOD_TLS_CLIENT;
+
+    /**
+     * Encryption: TLSv1.0.
+     *
+     * @const int
+     */
+    const ENCRYPTION_TLS_v1_0 = STREAM_CRYPTO_METHOD_TLSv1_0_CLIENT;
```

Comment:
Versioning format is not consistent with SSL constants (`ENCRYPTION_SSLv23` vs. `ENCRYPTION_TLS_v1_1`) but the first format is wrong. Is `23` 2.3 or 23? So, since SSL is outdated, I am suggesting to change the format for TLS and maintain the format for SSL untouched to keep BC. However, can change `TLS_v1_1` for `TLSv1_1`? We save one underscore and it reduces the delta with SSL format.

Do you agree?